main.go

```diff
@@ -1,11 +1,47 @@
 package main
```

Comment:
If we want to productionize it, it would be more standard to have an "Example"-like test. If we don't have an "Example" or an output assertion, the program should explain why is it necessary and what it should expect.

Reply:
main.go file is mostly for testing how things work, it won't be used later once we have this working. Regarding example test - I agree.